gwpy/plot/colorbar.py

```diff
@@ -69,24 +59,8 @@ def process_colorbar_kwargs(figure, mappable=None, ax=None, use_axesgrid=True,
     # -- create axes for colorbar (if required)
 
     cax = kwargs.pop('cax', None)
-    if cax is None:
-        # show warning about pending change
-        #     current default is to shrink the axes a wee bit,
-        #     which will change to no shrinking before 1.0
-        if kwargs.get('fraction') is None:
-            warnings.warn("`fraction` was not specified. Currently the "
-                          "default is `0.15`, matching the previous release "
-                          "behaviour, however, this will change to `0.0` in "
-                          "an upcoming release. To keep the current default, "
-                          "manually specify `fraction=0.15`",
-                          PendingDeprecationWarning)
-            kwargs['fraction'] = 0.15
-
-        # create axes if requesting not to resize the parent
-        if use_axesgrid:
-            cax, kwargs = make_axes_axesgrid(ax, **kwargs)
-    else:
-        kwargs.pop('fraction', None)  # don't need this if already have axes
+    if cax is None and use_axesgrid:
+        cax, kwargs = make_axes_axesgrid(ax, **kwargs)
```

Comment:
@duncanmmacleod Should the scenario of 'cax is None' and 'use_axesgrid' is False be handled in the code?  (I haven't been able to infer enough about what `cax` and `use_axesgrid` do to determine this myself.)

Reply:
Explicitly no. That condition falls back to the default `matplotlib` behaviour. I will update the documentation to better explain this.